internal/data/testdata/metric.go

```diff
@@ -606,6 +678,9 @@ func initMetric(m pdata.Metric, name string, ty pdata.MetricDataType) {
 		histo := m.DoubleHistogram()
 		histo.InitEmpty()
 		histo.SetAggregationTemporality(pdata.AggregationTemporalityCumulative)
+	case pdata.MetricDataTypeDoubleSummary:
+		summary := m.DoubleSummary()
+		summary.InitEmpty()
```

Comment:
Should we also set aggregation temporality to cumulative here?

Reply:
We actually removed the aggregation temporality for the Summary metric in the proto (in this [PR](https://github.com/open-telemetry/opentelemetry-proto/pull/227)). The idea was that temporality does not make sense in the context of Summary (as it is more of an interval, rather than a delta or cumulative measurement).